software/salmon/quant/main.nf

```diff
@@ -40,12 +41,20 @@ process SALMON_QUANT {
         input_reads = "-a $reads"
     }
 
-    def strandedness = meta.single_end ? 'U' : 'IU'
-    if (meta.strandedness == 'forward') {
-        strandedness = meta.single_end ? 'SF' : 'ISF'
-    } else if (meta.strandedness == 'reverse') {
-        strandedness = meta.single_end ? 'SR' : 'ISR'
+    def strandedness =  ''
```

Comment:
```suggestion
    def strandedness =  'A'
```

Reply:
Instead of `lib_type` being a strict boolean we can allow for it to be a string that can override the default behaviour more explicitly. This will also be quite useful if for example the pipeline infers the wrong strand and the user wants to overwrite it with e.g. `IU`, `ISF` etc

Reply:
Yes, actually makes much more sense this way!